bin/webpack-dev-server.js

```diff
@@ -372,11 +378,15 @@ function startDevServer(wpOpt, options) {
 		}));
 	}
 
-	const uri = domain + (options.inline !== false || options.lazy === true ? "/" : "/webpack-dev-server/");
+	uri = domain + (options.inline !== false || options.lazy === true ? "/" : "/webpack-dev-server/");
+
+	liftDevServer(compiler, options, uri);
+}
 
+function liftDevServer(compiler, options, uri) {
 	let server;
 	try {
-		server = new Server(compiler, options);
+		server = new Server(compiler, _.cloneDeep(options));
```

Comment:
Perhaps we could use `Object.assign` instead of pulling in the whole lodash dependency? I don't want to use lodash for this one use.